tensorboard/backend/event_processing/plugin_event_accumulator.py

```diff
@@ -57,7 +60,8 @@
 _TENSOR_RESERVOIR_KEY = "."  # arbitrary
 
 
-class EventAccumulator(object):
+
+class EventAccumulatorPlugin(EventAccumulator):
```

Comment:
Please do not rename this class. That would require refactoring folks who depend on it.